lib/Backend/Inline.cpp

```diff
@@ -4667,6 +4667,10 @@ Inline::MapFormals(Func *inlinee,
             {
                 break;
             }
+            if (instr->m_func != inlinee)
```

Comment:
>instr->m_func != inlinee [](start = 16, length = 24)

Does this mean there could be some instruction before inlinee->m_headInstr but still belongs to inlinee?